tools/src/post-mortem/Main.hs

```diff
@@ -18,7 +21,7 @@ main = do
             err $ "sample probability: " ++ show sampleProb
             err ""
             xs <- forM logDirs $ flip processLogDirOverview sampleProb
-            chart xs "times.png"
+            chart xs "times.svg"
```

Comment:
What's the story behind the change of this extension?

Reply:
it outputs an svg image, but calls it `.png`, and my image viewer refuses to open such mis-named files